packages/account/src/Components/file-uploader-container/file-uploader.tsx

```diff
@@ -77,7 +85,9 @@ const FileUploader = React.forwardRef(({ onFileDrop, getSocket }, ref) => {
                     }
 
                     // send files
-                    const uploader_promise = uploader.upload(processed_files[0]).then(api_response => api_response);
+                    const uploader_promise = uploader
+                        .upload(processed_files[0])
+                        .then((api_response: unknown) => api_response);
```

Comment:
Instead of using **unknown** can't we define types for the response?

Reply:
The return value is part of `@binary-com/binary-document-uploader`. That lib has no @types. Hence I have mentioned it as unknown